### DISCUSSION TITLE
GL/glu.h: No such file or directory

### DISCUSSION THREAD

**Discussion by nimeeshrathi** [InteractiveComputerGraphics/SPlisHSPlasH#297] [Q&A] [answered]
Hi Jan , 
I am very excited to try out your new surface tension , for that I am trying to install new version of splishsplash and getting error ( below image ) when try to make the package. It will be really helpful if you have any possible solution for that .
System :- Ubuntu 
![error](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH/assets/93661005/1d82be0c-35d6-407a-b6c1-6fbd7c35f679)

**Comment by janbender** [comment, marked as answer]
It seems that the OpenGL Utility Library is missing on your system.

**Reply by nimeeshrathi** [reply]
Thanks
